AU2/frontends/inquirer_cli.py

```diff
@@ -169,7 +170,7 @@ def render(html_component, dependency_context={}):
 
     # dependent component
     elif isinstance(html_component, AssassinPseudonymPair):
-        assassins = [a[0] for a in html_component.assassins]
+        assassins = list(set([a[0] for a in html_component.assassins] + [a for a in html_component.default.keys()]))

```

Comment:
This shouldn't actually be an issue, since the `assassins` passed to `AssassinPseudonymPair` by `CorePlugin`'s `on_event_request_update` do include any hidden assassins that are already in the event. (This is precisely the reason why the `include_hidden` argument of `AssassinsDatabase.get_ident_pseudonym_pairs` is a `Callable` and not a `bool`). But an extra layer of protection doesn't hurt...